perftracker/templates/comparison_id.html

```diff
@@ -364,7 +465,74 @@ <h4>Job: {{ job }}</h4>
     $(window).on('DOMContentLoaded load resize scroll', delayed_render);
     $('.collapse').on("shown.bs.collapse hidden.bs.collapse", delayed_render);
     delayed_render();
+
+    var keyUpStack = [];
+
+    $(".panel__input-search_text").keyup(function(e) {
+        keyUpStack.push(1);
+        $('h4').unmark();
+
+        setTimeout(function() {
+            keyUpStack.pop();
+
+            if(keyUpStack.length === 0) {
+                var buf = this.value;
+
+                var filteredList = {};
+                var plotsToShow = {};
+                Object.keys(plotsTitles).forEach(function(key) {
+                    if(!plotsTitles[key].replace(/\s/g, '').includes(buf.replace(/\s/g, ''))) {
+                        filteredList[key] = plotsTitles[key];
+                    }
+                });
+
+                Object.keys(plotsTitles).forEach(function(key) {
+                    if(plotsTitles[key].replace(/\s/g, '').includes(buf.replace(/\s/g, ''))) {
+                        plotsToShow[key] = plotsTitles[key];
+                    }
+                });
+
+                if(this.value.length >= 3) {
+                    Object.keys(filteredList).forEach(function(key) {
+                        sect_data[key].section_hidden = true;
+                        $('.graph__container_' + key).hide();
+                    });
+                } else if(e.keyCode === ENTER_KEY) {
+                    Object.keys(filteredList).forEach(function(key) {
+                        sect_data[key].section_hidden = true;
+                        $('.graph__container_' + key).hide();
+                    });
+                }
+
+                Object.keys(plotsToShow).forEach(function(key) {
+                    sect_data[key].section_hidden = false;
+                    $('.graph__container_' + key).show();
+                });
+                $('h4:not(.left-nav__headers)').mark(buf, {
```

Comment:
better to use explicit class for our titles, not exclude some random classes from 3rd party code.
If we add some new UI with h4 class, it still will be highlighted